src/databricks/labs/ucx/workspace_access/clusters.py

```diff
@@ -0,0 +1,117 @@
+import logging
+
+from databricks.labs.blueprint.installation import Installation
+from databricks.labs.blueprint.tui import Prompts
+from databricks.sdk import WorkspaceClient
+from databricks.sdk.errors import InvalidParameterValue
+from databricks.sdk.service.compute import ClusterDetails, DataSecurityMode
+
+logger = logging.getLogger(__name__)
+
+
+class ClusterAccess:
+    def __init__(self, installation: Installation, ws: WorkspaceClient, prompts: Prompts):
+        self._ws = ws
+        self._prompts = prompts
+        self._installation = installation
+
+    def list_cluster(self):
+        cluster_list = {}
+        clusters = self._ws.clusters.list()
```

Comment:
Use the Request class to filter the results on the remote side
`all = w.clusters.list(compute.ListClustersRequest())`
https://databricks-sdk-py.readthedocs.io/en/latest/workspace/compute/clusters.html

Reply:
@dmoore247 That class doesn't exist.